src/webview/MessageList.js

```diff
@@ -61,6 +64,18 @@ type SelectorProps = {|
   doNotMarkMessagesAsRead: boolean,
 |};
 
+/**
+ * All the data for rendering the message list, and callbacks for its UI actions.
+ *
+ * This data gets used for rendering the initial HTML, for computing
+ * inbound-events to update the page.  Then the handlers for the message
+ * list's numerous UI actions -- both for user interactions inside the page
+ * as represented by outbound-events, and in action sheets -- use the data
+ * and callbacks in order to do their jobs.
+ *
+ * This is also -- hence the name -- the React props for the inner, function
+ * component, which include data obtained through the various HOCs below.
+ */
```

Comment:
> ```
>  * This data gets used for rendering the initial HTML, for computing
>  * inbound-events to update the page. […]
> ```

Should the comma be replaced with "and"? It looks like this might've started its life as "A, B, and C" but then you removed ", and C". Is that right? 😛

Reply:
Hmm yeah, indeed. I think C was what turned into the separate next sentence.